configs/m2cellCfg.yaml

```diff
@@ -21,3 +21,15 @@ TelemetryServer:
   host: "127.0.0.1"
   port: 50001
   threads: 3
+
+# Global constants
+Globals:
+  # weight of the seconrary mirror. Units???
+  const_mirror_weight: 15140
```

Comment:
Definitely the unit is not kg. This is what I have in simulator:
https://github.com/lsst-ts/ts_m2com/blob/develop/python/lsst/ts/m2com/constant.py#L37

I guess this value is closed to 1588.65 kg * 9.8 m/s^3 = 15568.77 N. Please double check the right one. In addition, the unit of weight is not Newton.

Reply:
After talking to @pcortes80, The units should be Newtons, but the value is off.

Reply:
If the unit is Newton, I am afraid that maybe you should not use the `weight`.

Reply:
A Newton is a perfectly reasonable unit for weight.  It would be unacceptable for mass.

The names come directly out of the csv file for testing, where it really helps to have the names similar to what is in the LabView vi. It's already difficult to line everything up and adding more names would make it worse.

Reply:
It looks like the units should be Newtons, but you and Patricio have different values for the mirror mass. He has a theoretical value from Harris of 1545. You have 1588.65

Reply:
Added [DM-36737] to deal with constant names in the csv files and configuration files, as well as units, which appear to be Newtons. There seem to be conflicting values for the mass of the mirror.